robot/__init__.py

```diff
@@ -1,5 +1,7 @@
-from robot.robot import Robot
-from robot.motor import BRAKE, COAST
-from robot.game import GameMode
-from robot.game_specific import *
-from robot.servo import PinMode, PinValue
+"""Userspace API for a robot running ``robotd``."""
+
+from robot.robot import Robot  # noqa
+from robot.motor import BRAKE, COAST  # noqa
+from robot.game import GameMode  # noqa
+from robot.game_specific import *  # noqa
+from robot.servo import PinMode, PinValue  # noqa
```

Comment:
Rather than `noqa`, we should make the export explicit with `__all__`.

Reply:
The challenge is that that would then require duplicating all the game-specific definitions in this file. I'm not sure that's really something we want, though maybe is the best solution.

Reply:
If `__all__` is correctly evaluated, we could explicitly define everything else, and then unpack the variables from the `game_specific` module into `__all__`?

It's a hack, but it could work, and be _fairly_ nice?

Reply:
I don't think that's how `__all__` works -- you need to populate it with strings of symbol _names_.

Reply:
c9ef126 explicit is better than implicit. The overhead here is minimal (linting will keep us honest if we change things, though admittedly not if we add things), so I think this is best.